Src/VimCore/UndoRedoOperations.fs

```diff
@@ -363,7 +363,16 @@ and UndoRedoOperations
                 | UndoRedoData.Linked (count, false) :: tail -> 
                     Contract.Assert(count > 0)
                     hasData <- true
-                    _undoStack <- UndoRedoData.Linked (count, true) :: tail
+
+                    // Append an empty transaction to the previous undo transaction
+                    // to break the undo sequence.
+                    match _textUndoHistory with

```

Comment:
This is asserted above in the `if`. Can probably simplify those two checks.

Reply:
Fixed that.